src/__mocks__/View.js

```diff
@@ -9,7 +9,9 @@
 
 'use strict';
 
-var createReactNativeComponentClass = require('createReactNativeComponentClass');
+var createReactNativeComponentClass = require(
+  'createReactNativeComponentClass',
+);
```

Comment:
This is fixed in master, I'll do a new release tomorrow with those fixes:
https://github.com/prettier/prettier/commit/705ac7d3cf23e8b6f2cd612d5a3cdf2360a5c32c